### PR TITLE
Add form blog

### PR DESCRIPTION
**Pull request by ademibowale**
## In this blog app project 

- Adding forms to the Blog app.
- I use preprocessed HTML file with embedded Ruby code.
- I use layouts and templates for shared content.

## I made changes on  

- create html form for comments 
- modified the post comments

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root "users#index"
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show, :create, :new] do
+      resources :comments, only: [:create]
```
Comment (FrederickMih) [inline config/routes.rb]: - [x] Based on the requirements of this project,  there should be a pair of routes for creating comments:

`resources :comments, only: [:create, :new]`


Let's kindly fix :wrench: it.


- [x] In order for us to create a new comment, there should be a form in the views/comments/new.html.erb file which accept inputs for creating comments. Let's kindly fix :wrench: it.

Please this should not be done in the views/post/ directory. I saw a partial(_add_comment.html.erb) here for creating a new comment. This should be done properly in the directory shown above. This is a Rails great way of organizing our codebase for future reference.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @ademibowale, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!


Screenshot from 2022-10-20 09-38-51


## Highlights :green_circle: 

- Descriptive README.md file :+1: 
- Correct Gitflow :heavy_check_mark: 
- PR has a good title and summary :+1: 
- No Linter error :+1: 

## Required Changes ♻️


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (ademibowale) [reply config/routes.rb]: Thank sir i really appreciate you for taking much of your time to view my code ,great job and all the changes required have been fixed .

**Comment by ademibowale** [comment]
# In this blog app project 

## I made changes on  

- create html form for comments 
- modified the post comments

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @ademibowale,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**